test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
delete this and the `test/models/user_test.rb` files because you are not using them, you are using `rspec` (the `spec/` directory), WDYT?